metamorph/store/dynamodb/dynamodb.go

```diff
@@ -416,7 +473,7 @@ func (ddb *DynamoDB) GetBlockProcessed(ctx context.Context, blockHash *chainhash
 
 	var processedAtTime time.Time
 	if blockItem.ProcessedAt != "" {
-		processedAtTime, err = time.Parse(ISO8601, blockItem.ProcessedAt)
+		processedAtTime, err = time.Parse(time.RFC3339, blockItem.ProcessedAt)
```

Comment:
time.RFC3339Nano is probably better fit if you need nano precision as per original format. But up to you ;)